### ISSUE TITLE
XG 3.3.0.0 crashes on loading the webinterface: Error destroying handle 0x43b mutex due to 16

### ISSUE THREAD

**Issue by Strubbl** [lformella/xdcc-grabscher#73]
This is a follow-up issue after #72 had been fixed with.

When I start XG in the console and do nothing it seems to run forever. But when i open up a browser (tested with FF 33 and Chromium 38) and open the URL http://localhost:5556/ the following happens:
- xg redirects me to the URL http://localhost:5556/3.3.0.0/Resources/index.html
- xg loads the template website with the title "XG Webfrontend v{{VERSION}}"
- after a second the title changes to "XG Webfrontend v3.3.0.0"
- but browser window is still white
- so i have a look in the terminal and see XG crashed with the following message

```
~/opt/xg$ mono XG.Application.exe 
Error destroying handle 0x43b mutex due to 16

Abgebrochen (Speicherabzug geschrieben)
```

It is always reproducible with FF and Chromium. 
How can I enable logging and squeeze more information out?

**Comment by lformella** [comment]
Where can I get a pre installed virtual box image of your Arch Linux version full of mono exceptions?

Is this ok?
http://virtualboxes.org/images/archlinux/ - Archlinux 2013.08.01 (if the torrent is still alive)

**Comment by Strubbl** [comment]
I hope. If not, I am going to prepare a minimal image, where the problem can be reproduced.

**Comment by Strubbl** [comment]
It is my system. :( Not reproducible on a clean system. :) Sorry for bothering you. I need to find the problem in my installation. ATM I have no idea where the problem comes from.

(P.S.: but all other problems before are the same even in the clean installation, e.g. wrong Db4o DLLs, SharpRobin problem when having no config/starting first time)

**Comment by Strubbl** [comment]
haha, now i have started xg with strace and it works:

strace mono XG.Application.exe

WTF??? I can get to the web frontend. I don't understand the world anymore. Crazy stuff.

#edit
ok, some more insight. Not only does XG work with strace in front. Even with "time" in front it works. If i have the blank start of XG, I always get the same error message.

```
$ mono XG.Application.exe 
Error destroying handle 0x438 mutex due to 16
```

And i really mean the exact same error message. even the address 0x438 does not change anymore.

**Comment by squirrelsama** [comment]
@Strubbl I am experiencing this issue on Arch Linux x64 with mono versions 3.2-3.10. I run Arch at home and work.  Both machines were installed with the Dec 1 2014 Arch ISO. At home, I cannot reproduce (AMD CPU), at work (Intel Haswell CPU) I can. Even on a freshly wiped install today. Could CPU make have affect your ability to repro?

**Comment by Strubbl** [comment]
@squirrelsama maybe, i have no idea. i am using 64 bit arch linux, too.
